AmplifyPlugins/Core/AWSPluginsCoreTests/Model/GraphQLDocument/GraphQLCreateMutationTests.swift

```diff
@@ -16,6 +16,10 @@ class GraphQLCreateMutationTests: XCTestCase {
     override func setUp() {
         ModelRegistry.register(modelType: Comment.self)
         ModelRegistry.register(modelType: Post.self)
+        ModelRegistry.register(modelType: Project2V2.self)
+        ModelRegistry.register(modelType: Team2V2.self)
+        ModelRegistry.register(modelType: Record.self)
+        ModelRegistry.register(modelType: RecordCover.self)
```

Comment:
those models were missing, so the selection set in queries wasn't properly generated (see GraphQL queries below)